src/dl_patch_ptr.cpp

```diff
@@ -1,70 +1,48 @@
 #include "dl_patch_ptr.h"
```

Comment:
The changes in this file were needed to be able to patch pointers which comes from the default values added from the json parser when member data is missing.

I'm now looking at merging strings from that data as well, and have found that adding all that logic here as well isn't maintainable. So I will revert all the changes made in this file when I add string merging from json.

That said, this pull request is already too big to add the json string merging to it as well, so please just ignore the changes in dl_patch_ptr.h and cpp, pretend they aren't here.